### PR TITLE
Upgrade GRPC, BoringSSL and cloud-sink libraries

### PR DESCRIPTION
**Pull request by denyska**

### CODE REVIEW

File: collector/src/test/java/com/google/cloud/trace/zipkin/ZipkinCollectorIntegrationTest.java [hunk 1]
```diff
@@ -215,15 +220,24 @@ public Credentials mockGoogleCredentials() throws IOException {
...
           NettyChannelBuilder.forTarget(storageProperties.getApiHost())
               .sslContext(CLIENT_SSL_CONTEXT)
               .build();
-      TraceServiceGrpc.TraceServiceBlockingStub traceService =
-          TraceServiceGrpc.newBlockingStub(managedChannel)
-              .withCallCredentials(MoreCallCredentials.from(credentials));
-      final GrpcTraceConsumer traceConsumer = new GrpcTraceConsumer(traceService);
+
+      TransportChannelProvider channelProvider =
+          FixedTransportChannelProvider.create(GrpcTransportChannel.create(managedChannel));
```
Comment (adriancole) [inline collector/src/test/java/com/google/cloud/trace/zipkin/ZipkinCollectorIntegrationTest.java]: so was this part the secret of the incas? (ie the part that fixed the hung test thing)

Comment (denyska) [reply collector/src/test/java/com/google/cloud/trace/zipkin/ZipkinCollectorIntegrationTest.java]: yep, I ruled SSL out by adding main() to StackdriverMockServer and connecting to it with "openssl s_client -connect localhost:31891". Apparently new grpc server doesn't like credentials pushed unless they are expected.

**Comment by denyska** [comment]
@bogdandrutu jfyi

**Comment by adriancole** [comment]
Ahah. Thanks for taking time to figure this out.

**Comment by adriancole** [comment]
thx!

**Pull request merged**